include/zephyr/bluetooth/mesh/access.h

```diff
@@ -893,7 +893,7 @@ struct bt_mesh_model {
 	uint16_t * const groups;
 	const uint16_t groups_cnt;
 
-#if CONFIG_BT_MESH_LABEL_COUNT > 0
+#if (CONFIG_BT_MESH_LABEL_COUNT > 0) || defined(__DOXYGEN__)
```

Comment:
https://github.com/zephyrproject-rtos/zephyr/pull/58311#discussion_r1206587375
Otherwise it won't be rendered. Check the currently published doc and from this PR.